app/templates/node_info.html

```diff
@@ -225,14 +226,148 @@ <h3>{{ node.name }}</h3>
+                        <tr>
+                            <th>Name</th>
+                            <th>Size</th>
+                            <th>Storage Class</th>
+                            <th>Storage Phase</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for volume in pod.volume_details %}
+                        <tr>
+                            <td>{{ volume.pvc_name }}</td>
+                            <td>{{ volume.size }}</td>
+                            <td>{{ volume.class }}</td>
+                            <td>{{ volume.phase }}</td>
+                        </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </td>
+        </tr>
+        <tr>
+            <td>Potential Ready</td>
+            <td>{{ pod.potential_ready }}</td>
+        </tr>
+        {% if pod.affinity %}
+        <tr>
+            <td>Affinity</td>
+            <td>
+                <table>
+                    <thead>
+                        <tr>
+                            <th>Type</th>
+                            <th>Policy</th>
+                            <th>Details</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for key, value in pod.affinity.items() %}
+                        <tr>
+                            <td>{{ key }}</td>
+                            <td>{{ value.policy }}</td>
+                            <td>{{ value.details }}</td>
+                        </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </td>
+        </tr>
+        {% endif %}
+        <tr>
+            <td>Node Details</td>
+            <td>
+                <table>
+                    <thead>
+                        <tr>
+                            <th>Key</th>
+                            <th>Value</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for key, value in pod.node_details.items() %}
+                        <tr>
+                            <td>{{ key }}</td>
+                            <td>{{ value }}</td>
+                        </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </td>
+        </tr>
+    </tbody>
+</table>
 <script
         src="/static/libs/jquery-3.6.0.slim.min.js"
         integrity="sha256-u7e5khyithlIdTpu22PHhENmPcRdFiHRjhAuHcs05RI="
         crossorigin="anonymous"></script>
 <script type="text/javascript" charset="utf8" src="/static/libs/datatables.min.js"></script>
 <script>
     $(document).ready( function () {
-        $('#table').DataTable({
+        $('#table_substrate, #table_kubernetes').DataTable({
             // https://datatables.net/reference/option/
             "lengthMenu": [[-1], ["All"]]
```

Comment:
You can set a fixed size of the left column with:
```
            // https://datatables.net/reference/option/
            "lengthMenu": [[-1], ["All"]],
            "columnDefs": [
              { "width": "10%", "targets": 0 }
            ]
        });
```

Reply:
This is what I'm looking for! Thanks!